### ISSUE TITLE
Export propTypes of ReduxForm HOC

### ISSUE THREAD

**Issue by panayi** [erikras/redux-form#351]
Is it possible to export the [propTypes of ReduxForm](https://github.com/erikras/redux-form/blob/7bc159d7bb5795c9f89a4cc4ee9b420811b644a9/src/createHigherOrderComponent.js#L137) so that it is easy to add to the `WrappedComponent.propTypes`?

I'm using [react-strict-prop-types](https://github.com/gajus/react-strict-prop-types) which will make it far easier to just do:

```javascript
import { reduxForm, propTypes as reduxFormPropTypes } from 'redux-form'

@reduxForm({
  form: 'myForm',
  fields: ['firstName', 'lastName']
})
@StrictPropTypes()
export default class Form extends Component {
  static propTypes = {
     ...reduxFormPropTypes,
     // other local properties
  }
}
```

**Comment by erikras** [comment]
Good idea.

**Comment by erikras** [comment]
Released in [`v3.1.2`](https://github.com/erikras/redux-form/releases/tag/v3.1.2).

**Comment by panayi** [comment]
:+1:
